src/cargo/ops/cargo_install.rs

```diff
@@ -305,7 +305,12 @@ fn install_one(
     // *something* to install. Explicit `--bin` or `--example` flags will be
     // checked at the start of `compile_ws`.
     if !opts.filter.is_specific() && !pkg.targets().iter().any(|t| t.is_bin()) {
-        bail!("specified package `{}` has no binaries", pkg);
+        bail!(
+            "It's not possible to install package `{}`, because it has no binaries. \
```

Comment:
Conventionally Cargo's error messages start with lowercase letters and don't have a complete sentence to start out (e.g. no punctuation), but we do have errors which are followed up below with complete sentences and descriptions, could this perhaps match that style?

Reply:
I've reworded the message.